app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,51 @@
+class PasswordsController < ApplicationController
+  before_action :redirect_if_authenticated
+
+  def create
+    @user = User.find_by(email: params[:user][:email].downcase)
+    if @user.present?
+      if @user.confirmed?
+        @user.send_password_reset_email!
+        redirect_to root_path, notice: 'Reset instruction is sent to user emails'
+      else
+        redirect_to root_path, notice: 'Please confirm email first.'
```

Comment:
```suggestion
        redirect_to root_path, notice: 'Invalid Email!.'
```

Reply:
Bhaiya, this message is shown because when a user creates his account using the correct information, but he or she didn't confirm it by this email,.
Can I use this "Invalid Email!" for this purpose?

Reply:
Let it be as it is.